packages/client/tests/functional/issues/15204-conversion-error/tests.ts

```diff
@@ -7,20 +6,29 @@ declare let prisma: PrismaClient
 
 // Regression test for https://github.com/prisma/prisma/issues/15204
 testMatrix.setupTestSuite(
-  ({ providerFlavor }) => {
-    // TODO fails with "number must be an i64" or "number must be an i32" depending on the matrix
-    skipTestIf(providerFlavor === ProviderFlavors.JS_LIBSQL)('should return a descriptive error', async () => {
+  ({ providerFlavor, fieldType }) => {
+    test('should return a descriptive error', async () => {
       await prisma.$executeRaw`INSERT INTO "TestModel" ("id", "field") VALUES ("1", 1.84467440724388e+19)`
 
-      await expect(prisma.testModel.findMany()).rejects.toThrow(
-        expect.objectContaining({
-          code: 'P2023',
-          message: expect.stringContaining(
-            // Error message ends in two different strings depending on input: "... to `BigInt`" or "... to `Int`"
-            'Inconsistent column data: Could not convert from `BigDecimal(18446744072438800000)` to',
-          ),
-        }),
-      )
+      if (providerFlavor === undefined) {
+        await expect(prisma.testModel.findMany()).rejects.toThrow(
+          expect.objectContaining({
+            code: 'P2023',
+            message: expect.stringContaining(
+              `Inconsistent column data: Could not convert from \`BigDecimal(18446744072438800000)\` to \`${fieldType}\``,
+            ),
+          }),
+        )
+      } else {
+        await expect(prisma.testModel.findMany()).rejects.toThrow(
+          expect.objectContaining({
+            code: 'P2023',
+            message: expect.stringContaining(
+              `Conversion failed: number must be an integer in column 'field', got 1.84467440724388e19`,
```

Comment:
Proposal: add back the quotes wrapping the value to `prisma-engines` once the release is done. Will open an issue/PR for this.

Reply:
(No longer needed)